src/Microsoft.AspNetCore.SystemWebAdapters.Authentication/RemoteAuthenticationExtensions.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net.Http;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Authentication;
+
+/// <summary>
+/// Helper methods for registering remote authentication services
+/// </summary>
+public static class RemoteAuthenticationExtensions
+{
+    /// <summary>
+    /// Registers remote authentication auth handler with an authentication builder using a default scheme name.
+    /// </summary>
+    /// <param name="authenticationBuilder">The authentication builder to register the remote authentication handler with.</param>
+    /// <returns>The authentication builder updated with the remote authentication handler added.</returns>
+    public static AuthenticationBuilder AddRemote(this AuthenticationBuilder authenticationBuilder)
```

Comment:
We've been referring to it as `RemoteApp` for session - does it make sense to stay consistent there?